code/modules/surgery/surgery.dm

```diff
@@ -68,7 +68,7 @@
 
 /proc/spread_germs_to_organ(obj/item/organ/E, mob/living/carbon/human/user)
```

Comment:
Can make this a proc of organs, remove the first argument, and have it check for sterile before a parent call in the `internal` level.